src/test/java/com/mxf/rest/ArticleDAOTest.java

```diff
@@ -0,0 +1,58 @@
+package com.mxf.rest;
+
+import static org.junit.Assert.*;
```

Comment:
I prefer to use `AsserJ`: http://joel-costigliola.github.io/assertj/ instead of `org.junit.Assert`